public/js/monitor.js

```diff
@@ -226,29 +226,53 @@ const postData = (url, data = {}) => {
   .catch(error => console.error(error));
 };
 
+function showAdditionalBreaches(){
+  document.getElementById("show-additional-breaches").classList.toggle("hide");
+  const additionalBreaches = document.getElementById("additional-breaches");
+  additionalBreaches.classList.toggle("show-breaches");
+  //setting height this way enables transition easing... setting the new height to "auto" 
+  if (additionalBreaches.classList.contains("show-breaches")) {
+    additionalBreaches.style.height = additionalBreaches.scrollHeight + "px";
+  }
+}
+
+const handleRadioButtons = function(form) {
+  const inputFields = form.querySelectorAll(".radio-button-group, .button");
+  inputFields[0].querySelector("input").checked = true;
+  for (let x = 0; x < inputFields.length ; x++) { 
+    const input = inputFields[x];
+    input.addEventListener("focus", (e) => {
+      input.addEventListener("keydown", (e) => {
```

Comment:
Do we need to remove the `keydown` listener anywhere?
Like, if we focus the input 3 times, will it add the `keydown` event listener 3 times?